resources/lib/streamservice.py

```diff
@@ -216,6 +216,10 @@ def get_stream(self, video, roaming=False, api_data=None):
             # Get stream manifest url
             manifest_url = next(stream.get('url') for stream in stream_json.get('targetUrls') if stream.get('type') == protocol)
 
+            # External virtual subclip support (live2vod airdate feature)
+            if video.get('start_date') and video.get('end_date'):
+                manifest_url += '?t=' + video.get('start_date') + '-' + video.get('end_date')
+
             # Fix virtual subclip
             duration = timedelta(milliseconds=stream_json.get('duration'))
```

Comment:
When using `get()` I always think about, what would happen if `duration` is not found?
Would milliseconds=None work, or do we need to provide a working default?
```
$ python -c 'from datetime import timedelta; print(timedelta(milliseconds=None))'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: unsupported type for timedelta milliseconds component: NoneType
```

Reply:
Same for the other `get()` calls obviously.